src/current/_includes/releases/v23.1/v23.1.4.md

```diff
@@ -0,0 +1,67 @@
+## v23.1.4
+
+Release Date: June 20, 2023
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v23-1-4-security-updates">Security updates</h3>
+
+- Introduced the [`sql.auth.createrole_allows_grant_role_membership.enabled`](../v23.1/cluster-settings.html#setting-sql-auth-createrole-allows-grant-role-membership-enabled) cluster setting, which defaults to `false`. If it is set to `true`, then users who have the [`CREATEROLE` role option](../v23.1/create-user.html#role-options) can grant and revoke role membership in any non-admin role. A future major version of CockroachDB will remove this cluster setting and make this the default behavior. [#104445][#104445]
+
+<h3 id="v23-1-4-{{-site.data.products.enterprise-}}-edition-changes">{{ site.data.products.enterprise }} edition changes</h3>
+
+- Fixed an initialization race condition in [changefeed schema feeds](../v23.1/changefeed-examples.html) that could result in null pointer exceptions. [#104934][#104934]
+
+<h3 id="v23-1-4-sql-language-changes">SQL language changes</h3>
+
+- Users with [`MODIFYSQLCLUSTERSETTING`](../v23.1/security-reference/authorization.html#supported-privileges) will no longer be able to view non-`sql.defaults` [cluster settings](../v23.1/cluster-settings.html). [#104542][#104542]
```

Comment:
I don't fully understand this and it seems important. Does this mean that even though you have permission to modify the setting, you can't see settings that have already been changed from their default?